model-mesh/manager/manager.yaml

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: modelmesh-controller
+  labels:
+    control-plane: modelmesh-controller
+spec:
+  selector:
+    matchLabels:
+      control-plane: modelmesh-controller
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: modelmesh-controller
+        # Required in Watson Cloud envs to allow outbound access to etcd
+        network-policy: allow-egress
+    spec:
+      affinity:
```

Comment:
Looking at the affinity reminded me -- we should add pod anti-affinities for all the model mesh pods so they try deploying on different nodes if possible

Reply:
Example PR: https://github.com/red-hat-data-services/odh-manifests/pull/179